defaults/main.yml

```diff
@@ -13,5 +13,13 @@ letsencrypt_certbot_verbose: true
 letsencrypt_pause_services: []
 # Force Certificate Reneval
 letsencrypt_force_renew: true
-# Additional certbot arguments
-letsencrypt_certbot_args: []
+# certbot arguments
+letsencrypt_certbot_args: ["certonly",
+   "--text",
+   "-n",
+   "--no-self-upgrade",
+   "-m {{ letsencrypt_email }}",
+   "--domains {{ _letsencrypt_domains | default(letsencrypt_domain) }}",
+   "--agree-tos",
+   "--standalone",
+   "--expand"]
```

Comment:
I kind of prefer the `["foo", "bar"]` syntax myself over the more common (?) way of doing lists in YAML, I think for cli params it is a bit clearer, less confusing looking `-`'s around :)